presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/Launcher.java

```diff
@@ -155,7 +155,10 @@ protected Extensions getExtensions()
         @Override
         protected Object[][] getContents()
         {
-            return new Object[][]{{"project.version", readProjectVersion()}};
+            return new Object[][] {
+                    {"project.version", readProjectVersion()},
+                    {"product-tests.module", "presto-product-tests"},
+                    {"server.module", "presto-server"}};
```

Comment:
Can we format this like
```java
return new Object[][] {
        {"server.module", "presto-server"},
};
```
That will allow easily adding new values. I confirmed that it passes the style checker.